composer.json

```diff
@@ -4,13 +4,17 @@
     "license": "MIT",
     "type": "phpcodesniffer-standard",
     "require": {
-        "slevomat/coding-standard": "^4.8.6",
+        "slevomat/coding-standard": "^7",
         "squizlabs/php_codesniffer": "^3.2"
     },
     "scripts": {
         "ci": [
             "@composer validate --no-check-all --strict",
-            "xmllint -schema /tmp/phpcs.xsd src/ruleset.xml --noout"
+            "xmllint -schema /tmp/phpcs.xsd src/ruleset.xml --noout",
+            "phpcs  --standard=src/ruleset.xml -e"
```

Comment:
Tohle dělá describe toho standardu, takže si to načte všechny rules. Když nějaké neexistuje, tak to failne. Takže je to dobré při upgradu, jestli nezbylo nějaké smazaný sniff